acrobat/styles/styles.css

```diff
@@ -60,6 +60,12 @@ div [class *= PdfnowLifecycle__dropzone2___] {
   z-index: 1;
 }
 
+#CID h1, #CID [class^="DropZoneText__dropzoneDescription"],
+#CID [class^="DropZoneFooter__signInText"],
+#CID [class^="LifecycleDropZone__logoText"] {
+  font-family: 'Adobe Clean', adobe-clean,'Source Sans Pro',-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,sans-serif;
```

Comment:
can we fix the font-family globally and not just on individual elements?

Reply:
I guess better idea is added to the dc-core.css, what do you think?

.FontFamily {
    font-synthesis: weight;
    font-family:  **'Adobe Clean',** adobe-clean,Source Sans Pro,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Ubuntu,Trebuchet MS,Lucida Grande,sans-serif
}

Reply:
And that's coming from React Spectrum, so it's not something we can change easily.  Is it possible to get typekit to recognize both?

Reply:
Try many ways, this one is the more general that I found to override it

```
[class^="LifecycleDropZone"],
[class^="DropzoneContent"],
[class^="DropZoneFooter"] {
  font-family: 'Adobe Clean', adobe-clean , 'Source Sans Pro', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
}
```

Reply:
How about `#CID *`

Not sure which is more expensive.

Reply:
Seems like targeting specific tags (e.g., #CID div, p) is more efficient